src/connector/src/sink/doris_connector.rs

```diff
@@ -0,0 +1,412 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use core::mem;
+use core::time::Duration;
+use std::collections::HashMap;
+
+use base64::engine::general_purpose;
+use base64::Engine;
+use bytes::{BufMut, Bytes, BytesMut};
+use http::request::Builder;
+use hyper::body::{Body, Sender};
+use hyper::client::HttpConnector;
+use hyper::{body, Client, Request, StatusCode};
+use hyper_tls::HttpsConnector;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use tokio::task::JoinHandle;
+
+use super::{Result, SinkError};
+
+const BUFFER_SIZE: usize = 64 * 1024;
+const MIN_CHUNK_SIZE: usize = BUFFER_SIZE - 1024;
+pub struct DorisInsertClient {
+    url: String,
+    header: HashMap<String, String>,
+    sender: Option<Sender>,
+}
+impl DorisInsertClient {
+    pub fn new(url: String, db: String, table: String) -> Self {
+        let url = format!("{}/api/{}/{}/_stream_load", url, db, table);
+        Self {
+            url,
+            header: HashMap::default(),
+            sender: None,
+        }
+    }
+
+    pub fn set_url(mut self, url: String) -> Self {
+        self.url = url;
+        self
+    }
+
+    pub fn add_common_header(mut self) -> Self {
+        self.header
+            .insert("expect".to_string(), "100-continue".to_string());
+        self
+    }
+
+    pub fn set_label(mut self, label: String) -> Self {
```

Comment:
This method is unsed, meaning that doris will generate a uuid for it. However, I think we need to set the label if we support exactly once via 2pc in the future. Please leave a comment to suggest future usage.